src/renderer/components/InputCurrency.js

```diff
@@ -212,7 +213,9 @@ class InputCurrency extends PureComponent<Props, State> {
         onChange={this.handleChange}
         onFocus={this.handleFocus}
         onBlur={this.handleBlur}
-        renderRight={renderRight || this.renderListUnits()}
+        renderRight={
+          renderRight || (this.props.doNotRenderListUnits ? this.renderListUnits() : null)
```

Comment:
Wouldn't it be `(!this.props.doNotRenderListUnits ? this.renderListUnits() : null)`? and perhaps extract it so we don't pass it down inside `...rest`?